check/setup.go

```diff
@@ -328,35 +352,13 @@ func (check *HealthCheck) closeConnection(deleteTopicIfPresent bool) error {
 	return nil
 }
 
-func (check *HealthCheck) deleteTopic(zkConn ZkConnection, chroot, name string, partitionID int32) error {
-	log.Infof("removing from topic %s", name)
-	topic := ZkTopic{Name: name}
-	err := zkPartitions(&topic, zkConn, name, chroot)
+func (check *HealthCheck) deleteTopic(name string) error {
+	acceptableTimeout := 3 * time.Second
+	err := check.broker.DeleteTopic([]string{name}, acceptableTimeout)
```

Comment:
here we cannot just replace the logic for shrinking the number of partitions of the replication topic with its deletion.
The idea here is that the topic has to be deleted only when the amount of replicas in it is `< 2` so we cannot just get rid of that part of the logic.